docs/fast_data/what_is_fast_data.md

```diff
@@ -4,61 +4,132 @@ title: What is Fast Data
+Business is changing rapidly with companies aiming to provide seamless digital experiences across numerous touchpoints and applications. APIs are crucial in this scenario, as they enable data exposure from different sources to various channels.
 
 ![Fast Data overview](./img/fastdata-overview.png)
 
-The main goal of Fast Data is to aggregate business data from different sources (that we call **Systems of Record**) into a single MongoDB collection called **Single View**. This collection can be easily queried by your APIs. The aggregation is performed only when changes occur to the source data, making your read operations blazing fast as the data is already there.
+Mia-Platform Fast Data let you to decouple your data from different sources, that we call **Systems of Record** (SoRs), aggregate them into a single MongoDB collection called **Single View** than can be easily queried by your APIs and thus build your out-of-the-box **Digital Integration Hub** (DIH). The consolidation is performed only when the source data changes, ensuring read operations are blazing fast since the data is pre-aggregated.
+
+The aggregation is performed using an important intermediary collection known as **Projection**. This collection is a standardized version of the original data, featuring only the fields of interest and *casting* them into a common format. For example, while your data sources might use various date formats, proper configuration allows you to unify them into a single format used across all Projections.
 
-The aggregation is done using another, valuable, intermediary collection called **Projection**. This collection is a **standardized** version of the original data, containing only the fields you are interested in, and *casting* them to a common format (e.g. it is common to have multiple date formats in your data sources, but with the correct configuration you can solve this issue and have one format shared among all Projections).
+Additionally, Mia-Platform Fast Data offers pre-built microservices and a user-friendly interface for setting up data flows in a low-code manner. This approach allows you to get started quickly and efficiently.
 
-Moreover, Fast Data provides ready-to-use microservices and a user-friendly interface to set up the data flow in a **low-code** fashion. The low-code configuration helps you to get started in a very short amount of time.
+Visit **[this page](./concepts/architecture.md)** to learn more about the architecture of Mia-Platform Fast Data.
 
-## How Fast Data can impact your business
+## Why Fast Data
 
-In this section we present the main scenarios in which Fast Data can significantly boost your company.
+Mia-Platform Fast Data combines **Data Mesh** and **Data Fabric** strategy, leveraging a suite of highly configurable tools that integrate the advantages of **Internal Developer Platform** while operationalizing your own **Digital Integration Hub**.
```

Comment:
Here I would describe briefly what `Data Mesh` and `Data Fabric` strategies represents

Reply:
Hi Giovanni, below my proposals:

**Data Mesh** architecture is a decentralized alternative to central data lakes. It organizes data by specific business domains, giving domain teams ownership and accountability for their data, which enhances data quality and governance. Data Mesh allows teams to independently scale their data systems using the technologies and tools that best fit their needs.
**Data Fabric** is an adaptive, flexible, and secure integrated data architecture that provides a unified view of data across various sources, addressing challenges faced by data lakes. It simplifies data integration, transformation, and movement, allowing for faster and more efficient data access

Reply:
Hi @crucio14 

Sounds good to me! :+1: